deno/payloads/v10/_interactions/applicationCommands.ts

```diff
@@ -70,6 +70,14 @@ export interface APIApplicationCommand {
 	 * The parameters for the `CHAT_INPUT` command, max 25
 	 */
 	options?: APIApplicationCommandOption[];
+	/**
+	 * Set of permissions represented as a bitset
+	 */
+	default_member_permissions?: Permissions;
```

Comment:
```suggestion
	default_member_permissions?: Permissions | null;
```
https://github.com/discord/discord-api-docs/pull/4830/commits/0dc91010bb37ed059a03a6ec52f6dc110ff3b684